### PR TITLE
Display URLs for tracking exited validators

### LINKED ISSUE #7208: [Tracking Issue] Inform how the user can track their validator exit status
# 💎 Issue

### Background

Follow-up to #6882 

After submitting a voluntary exit, the user is not able to track the status of his exit in the Prysm client. It would be nice to log a link to some block explorer's web page where the status of the exit would be displayed, e.g. https://beaconcha.in/exits/0x29389283982 

### Description

~~- [ ] Have the functionality available on a block explorer's website.~~
~~- [ ] Add a log with the correct link to the block explorer after a successful exit has been submitted.~~

### PR DESCRIPTION
**Pull request by rkapka**
**What type of PR is this?**

Feature

**What does this PR do? Why is it needed?**

Design for voluntary exits included informing users how they can track the status of their exit. Ideally we wanted to include a URLs to an exit-dedicated subpage on beaconcha.in, but this functionality is not a top priority for them. Therefore we settled for displaying the validator's URL on beaconcha.in. 

**Which issues(s) does this PR fix?**

Fixes #7208 

**Other notes for review**

N/A

### CODE REVIEW

File: validator/accounts/accounts_exit.go [hunk 1]
```diff
@@ -222,9 +220,38 @@ func performExit(cliCtx *cli.Context, cfg performExitCfg) ([]string, error) {
...
-	return formattedExitedKeys, nil
+	return rawExitedKeys, formattedExitedKeys, nil
+}
+
+func displayExitInfo(rawExitedKeys [][]byte, trimmedExitedKeys []string) {
+	if len(rawExitedKeys) > 0 {
+		urlFormattedPubKeys := make([]string, len(rawExitedKeys))
+		for i, key := range rawExitedKeys {
+			var baseUrl string
+			if params.BeaconConfig().ConfigName == "pyrmont" {
```
Comment (terencechain) [inline validator/accounts/accounts_exit.go:236]: Let's define this in the `param` pkg via a map to avoid fragmentation down the line

Comment (rauljordan) [reply validator/accounts/accounts_exit.go:236]: agreed

**Review by rauljordan — COMMENTED** [review]
Same comment as Terence about putting that block explorer string somewhere in our shared params package

File: shared/params/values.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const (
+	Mainnet configName = iota
+	EndToEnd
+	Pyrmont
+	Toledo
+)
+
+// ConfigNames provides network configuration names.
+var ConfigNames = map[configName]string{
+	Mainnet:  "Mainnet",
```
Comment (rauljordan) [inline shared/params/values.go:12]: Should these be consistent lowercase?

**Review by rauljordan — APPROVED** [review]

File: shared/params/BUILD.bazel [hunk 3]
```diff
@@ -16,6 +16,7 @@ go_library(
         "testnet_pyrmont_config.go",
         "testnet_toledo_config.go",
         "testutils.go",
+        "values.go",
```
Comment (terencechain) [inline shared/params/BUILD.bazel:19]: I'd probably rename this to `testnet_values.go`

Comment (terencechain) [reply shared/params/BUILD.bazel:19]: or `network_values.go`

**Pull request merged**